src/Uno.UITest.Helpers/AppInitializer.cs

```diff
@@ -268,7 +268,7 @@ private static IApp CreateiOSApp(bool alreadyRunningApp)
 
 			var app = alreadyRunningApp
 				? iOSConfig.ConnectToApp()
-				: iOSConfig.StartApp(Xamarin.UITest.Configuration.AppDataMode.DoNotClear);
+				: iOSConfig.StartApp(mode);
```

Comment:
⚠️ This may be incorrect. ⚠️ 

Actually from https://docs.microsoft.com/en-us/dotnet/api/xamarin.uitest.configuration.iosappconfigurator.startapp?view=xamarin-uitest-sdk docs, it says that AppDataMode is only supported in Xamarin Test Cloud. So it might not have any effect in this code path (and the android method above as well)?

Reply:
We may need to add a UI test for this then, which can be tricky to validate because it's order specific. Are you able to validate this locally ?

Reply:
> Are you able to validate this locally ?

No :(

No idea how. Would you be able to guide me through what's needed?

Reply:
You can follow this: https://github.com/unoplatform/Uno.UITest#running-the-tests-for-android

Reply:
@Youssef1313 Were you able to test it please?

Reply:
Oops sorry I missed this one. I'll finish some PRs and get back to this.

Reply:
Sorry for the delay here. I pushed my trial commit, but it doesn't work. I believe there is something (probably very fundamental) I'm not understanding properly.